docs/docs/manual/wikidata.md

```diff
@@ -6,28 +6,240 @@ sidebar_label: Wikidata
+To fetch the country code from an item representing a city, you need to follow two properties. First, follow [country (P17)](https://www.wikidata.org/wiki/Property:P17) to get to the item for the country in which this city is located, then follow [ISO 3166-1 alpha-2 code (P297)](https://www.wikidata.org/wiki/Property:P297) to get the two-letter code string.
+
+This is supported by the reconciliation interface, with a syntax inspired by [SPARQL property paths](https://www.w3.org/TR/sparql11-property-paths/): include the “country” column, and for <span class="menuItems">As Property</span> enter the sequence of property identifiers separated by slashes: “P17/P297.”
+
+This additional information can allow OpenRefine to disambiguate namesakes, at least to the country level. “Cambridge, US” is still ambiguous, so there will be multiple items with a perfect matching score, but “Oxford, GB” successfully disambiguates one particular city from other “Oxfords.” 
+
+The endpoint currently supports two property combinators: /, to concatenate two paths as above, and |, to compute the union of the values yielded by two paths. Concatenation / has precedence over disjunction |. The dot character . can be used to denote the empty path. For instance, the following property paths are equivalent:
+*   P17|P749/P17
+*   P17|(P749/P17)
+*   (.|P749)/P17
+
+They fetch the [country (P17)](https://www.wikidata.org/wiki/Property:P17) of an item or that of its [parent organization (P749)](https://www.wikidata.org/wiki/Property:P17).
+
+### Terms: labels, descriptions, aliases, and sitelinks
+
+Entities on Wikidata have what they call “terms:” labels, descriptions, aliases, and sitelinks. Each entity has human-readable preferred names (labels) in one or more languages. Each label comes with a description field, and a space for alternative labels in that language. Entities also can have a link to the related Wikipedia page in one or more langauges. 
+
+For example, [Q5](https://www.wikidata.org/wiki/Q5) has the English label “human” and the aliases “person” and “people,” with a description including “common name of Homo sapiens;” it also has the Italian label “umano” and the alias “persona” with a description “specie a cui appartiene il genere umano.” The English sitelink is “https://en&#46;wikipedia.org/wiki/Human” but Italian Wikipedia has no equivalent article attached to Q5. Not every language will have values, but these terms can still be useful for reconciling. 
+
+After reconciliation, you can extend your data with these terms. You can fetch aliases for a given term, or labels in another languages, by using <span class="menuItems">Edit column</span> → <span class="menuItems">Add columns from reconciled values....</span>. 
+
+You can refer to a term in a specific language by manually entering in a three-letter code. The first letter is the term (“L” for label, “D” for description, “A” for aliases, “S” for sitelink) and the next two are the language code. For example:
```

Comment:
This seems very obscure. I know that's not the docs fault, but perhaps we should see if we can improve it.